src/QMCDrivers/WFOpt/LinearMethod.cpp

```diff
@@ -20,9 +20,116 @@
 #include <vector>
 #include "QMCCostFunctionBase.h"
 #include <CPU/BLAS.hpp>
+#include "Numerics/MatrixOperators.h"
+#include "Numerics/DeterminantOperators.h"
```

Comment:
What is this needed for?

Reply:
MatrixOperators::product and invert_matrix